src/main/java/edu/umich/its/cpm/BoxUtils.java

```diff
@@ -172,7 +172,7 @@ public static void authenticate(String boxAPIUrl, String boxClientId,
 		}
 		catch (Exception e)
```

Comment:
I have change this exception to be IllegalArgumentException according to the CRUDRepository API.